### ISSUE TITLE
Installing a plugin with --link does not links the javascript file.

### ISSUE THREAD

**Issue by distante** [apache/cordova#161]
# Bug Report

## Problem
When developing a plugin, if the plugin is installed using the `--link` flag each change in the source Java File is applied to the plugin on next build (the file is a symlink). The same does not happens with the JavasSript file.

### What is expected to happen?
Te JavaScript file should also be linked

### What does actually happen?
An old javascript file lives in `platforms\android\platform_www\plugins\cordova-plugin-some-name\www\SomePlugin.js`

### Command or Code

cordova install some-plugin --link

### Environment, Platform, Device
Tested in Windows 10 with 8.1.2 (cordova-lib@8.1.1)

## Checklist

- [x] I searched for existing GitHub issues
- [x] I updated all Cordova tooling to most recent version
- [x] I included all the necessary information above

**Comment by brodybits** [comment]
A [minimal, reproducible example][minimal-reproducible-example] is needed.

[minimal-reproducible-example]: https://stackoverflow.com/help/minimal-reproducible-example

**Comment by breautek** [comment]
Yes this is more a feature request than a bug, I'm pretty sure linking js files was something that was never supported.

> Perhaps this has not been possible because the JavaScript gets modified by Cordova when it is pulled in to the final project.

I'm pretty sure this is the case, because we copy the JS file contents and insert it into another js file wrapped in a function, or something to that effect. This comes from ancient times before there were better bundling tools. We could consider using something such as rollup instead, and that may allow us to allow linking to js files and other assets. This was discussed at some point before but I couldn't find the email thread.
